### ISSUE TITLE
gb build all fails using go version 1.13.8

### ISSUE THREAD

**Issue by corbolais** [netzsinus/defluxio-software#24]
**TL;DR:** Need crosscompilation working? Use https://golang.org/dl/go1.6.4.linux-amd64.tar.gz

The `gb` tool fails to build this project using go version 1.13.8. A `gb` issue hints that go version < 1.10 is working.

And it does.

Just download the latest go version ***earlier than*** version 1.10: https://golang.org/dl/go1.9.7.linux-amd64.tar.gz
Unpack, set your PATH and verify your go version:
```
23:50 $ go version
go version go1.9.7 linux/amd64
```

Then go the usual `gb` path with `gb build all` and you're done.

One might verify the latest working go version >1.10, if any.

Edit: Crosscompilation fails even with go1.9.7, go1.6.4 seems to do the trick.

HTH

NB: Looks like noone cared so much as to build the defluxio-software project for a couple of years. I'd have assumed an earlier issue mentioned this, then.
